src/components/duet-date-picker/date-picker-day.tsx

```diff
@@ -35,45 +35,20 @@ export const DatePickerDay: FunctionalComponent<DatePickerDayProps> = ({
     onDaySelect(e, day)
   }
 
-  if (disabled) {
-    return (
-      <span
-        class={{
-          "duet-date__day": true,
-          "is-outside": isOutsideRange,
-          "is-disabled": disabled,
-          "is-today": isToday,
-          "is-month": isMonth,
-        }}
-        role="button"
-        tabIndex={isFocused ? 0 : -1}
-        onKeyDown={onKeyboardNavigation}
-        aria-pressed="false"
-        aria-disabled="true"
-        ref={el => {
-          if (isFocused && el && focusedDayRef) {
-            focusedDayRef(el)
-          }
-        }}
-      >
-        <span aria-hidden="true">{day.getDate()}</span>
-        <span class="duet-date__vhidden">{dateFormatter.format(day)}</span>
-      </span>
-    )
-  }
-
   return (
     <button
       class={{
         "duet-date__day": true,
         "is-outside": isOutsideRange,
         "is-today": isToday,
         "is-month": isMonth,
+        "is-disabled": disabled,
       }}
       tabIndex={isFocused ? 0 : -1}
       onClick={handleClick}
       onKeyDown={onKeyboardNavigation}
-      disabled={isOutsideRange}
+      aria-disabled={disabled ? "true" : undefined}
```

Comment:
Using `aria-disabled` only and not setting the `disabled` attribute breaks progressive enhancement. The button is still selectable when the date picker is rendered on the server and the client has no JavaScript. That's the reason for the `span` I've added.

(The `aria` rules on the `span` should present it to screen readers as a button to keep consistency. Just like we're styling it with css to look the same.)

Reply:
I think the progressive enhancement point is moot since you cant open the picker without javascript running on the client, so you can never attempt to interact with the aria-disabled buttons. Unless I'm misunderstanding?

Reply:
Yes, that is true. My bad. I just disabled css to "simulate" the server side rendered html 🙈
supporting progressive enhancement would then be another issue.